app/javascript/packages/address-search/package.json

```diff
@@ -1,25 +1,33 @@
 {
   "name": "@18f/identity-address-search",
-  "private": true,
-  "version": "1.0.0",
+  "version": "1.0.3",
   "type": "module",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/18f/identity-idp.git",
-    "directory": "app/javascript/packages/address-search"
-  },
+  "private": false,
   "files": [
-    "index.tsx"
+    "dist"
   ],
+  "scripts": {
+    "prepublishOnly": "webpack"
+  },
+  "exports": {
+    ".": {
+      "source": "./index.tsx",
+      "default": "./dist/index.js"
+    }
+  },
   "license": "CC0-1.0",
   "bugs": {
     "url": "https://github.com/18f/identity-idp/issues"
   },
   "homepage": "https://github.com/18f/identity-idp",
-  "dependencies": {
-    "swr": "^2.0.0"
-  },
   "peerDependencies": {
-    "react": "^17.0.2"
+    "focus-trap": "^6.7.1",
+    "swr": "^2.0.0",
```

Comment:
It's a bit of a gut call, but not sure we want _all_ the dependencies to be defined as peer dependencies, since it means the usage of the package would require the consumer to explicitly install those separately as well. This makes sense for React (and maybe for `swr`?), but not sure _all_ of them. At the very least, we'd probably want to be pretty loose with version ranges for all of the ones that are peer dependencies, unless we have a specific reason it must be `>= 6, < 7` & `>= 2 & < 3` for `focus-trap` and `swr` respectively.

This is also the sort of thing I might expect to be described in README installation instructions, similar to...

https://github.com/18F/identity-idp/blob/ac0012febfd133ad5f946e0e0e1a83e09dc609a4/app/javascript/packages/eslint-plugin/README.md#L9-L13

Reply:
@allthesignals In React's particular case, this is not just a matter of bloat - it's actually required for React to work as expected (e.g. when [multiple versions interact](https://stackoverflow.com/questions/66488492/solve-having-more-than-one-copy-of-react-in-the-same-app)).

If you expect that the library or dependency is going to act in isolation, then bloat may be the only major concern.

Reply:
> (Separately, for my own information: I don't understand how the "peer dependency" designation differs from simply _loosely_ versioning the dependency... wouldn't that have the same effect? I'll do some more reading but this has stuck out to me...)

If I recall, the default behavior of npm isn't very eager about trying to avoid different versions of the same package even if multiple packages would be satisfied by a single version, though there is [npm dedupe](https://docs.npmjs.com/cli/v8/commands/npm-dedupe#description) and the `prefer-dedupe` configuration which can help. Bundlers may operate differently, but even then I'd expect it differs between bundlers, so wouldn't want to make too strong an assumption if it was a package that'd be used by a broad set of consumers (not really a concern for us, yet).